roles/post/tasks/main.yml

```diff
@@ -2,12 +2,19 @@
 # roles/post/tasks/main.yml
 - name: add nginx reverse proxy for webui
   import_tasks: nginx.yml
+  delegate_to: "{{ groups['couchbase_cluster'][0] }}"
```

Comment:
`"{{ groups['couchbase_cluster'][0] }}"` can just be `localhost` to clean it up a bit.